.eslintrc

```diff
@@ -9,7 +9,6 @@
     "arrow-spacing": [2],
     "babel/arrow-parens": [2, "as-needed"],
     "brace-style": [2, "1tbs", {"allowSingleLine": true}],
-    "camelcase": [2, {"properties": "always"}],
```

Comment:
I know that situation. I used to disable it where necessary. There’s also the [possibility to disable it for property names](http://eslint.org/docs/rules/camelcase#never).